bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,149 @@
+import datetime
+import json
+import logging
+from pathlib import Path
+from typing import List, Tuple
+
+import discord
+import requests
+from discord.ext.commands import Bot, Cog, Context, command
+
+from bot.pagination import ImagePaginator
+
+
+logger = logging.getLogger(__name__)
+# RESTful API to get countries info
+url = "https://restcountries.eu/rest/v2/alpha/"
+# white flag image to show during error
+white_flag = "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/\
+White_flag_waving.svg/158px-White_flag_waving.svg.png"
+# url to get images of flags
+flag_url = "https://www.countryflags.io/"
+
+
+class NationDay(Cog):
+    """NationDay Cog that contains nationday command."""
```

Comment:
You can see that the this is a `NationDay(Cog)` on the line above, and the command is just below. There's no purpose to repeating what the surrounding code already communicates just for the sake of having *some* documentation - this is your opportunity to explain, in your own words, why the code is here, what it does, and how.